README.md

```diff
@@ -13,6 +13,14 @@ of vulnerability reports.
 This project is developed by [Trail of Bits](https://www.trailofbits.com/) with
 support from Google. This is not an official Google product.
 
+- [Installation](#installation)
+- [Usage](#usage)
+- [Examples](#examples)
+- [Security Model](#security-model)
+- [Licensing](#licensing)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
```

Comment:
This won't work on PyPI: https://github.com/pypa/readme_renderer/issues/169

Reply:
440029c